src/shared/util.js

```diff
@@ -80,6 +82,25 @@ function isSecure() {
   return location.protocol === 'https:';
 }
 
+/**
+ * Detect browser name and version.
+ * @return {Object} Browser name and major, minor and patch versions. Object is empty if info can't be obtained.
+ */
+function detectBrowser() {
+  const { name, version } = detect();
+  if (name === undefined || version === undefined) {
```

Comment:
This check does not work as expected because `detect()` returns `Object` or `null`.
And if `null` returned, the former line ends up with syntax error...

> https://github.com/DamonOehlman/detect-browser/blob/master/index.js#L48

but basically, we expect this code runs in browser env only, then all env returns `Object`.
So we can skip this check, I think.

Reply:
I amended + force pushed